react-web/src/app/core/pages/home/index.tsx

```diff
@@ -1,13 +1,30 @@
-import React from 'react'
+import { useForm } from 'react-hook-form'
+
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
 
 import { HomeTemplate } from 'components/templates/home'
 
-import logo from 'app/core/resources/logo.svg'
+const Home = (): JSX.Element => {
+  const validationSchema = yup.object().shape({
+    amountReceived: yup.number().min(1).required('This field is required.'),
+    amountSent: yup.number().min(1).required('This field is required.'),
+  })
+
+  const defaultValues: { amountReceived?: number; amountSent?: number } = {
+    amountReceived: 0,
+    amountSent: 0,
+  }
 
-import styles from './styles.module.scss'
+  const { handleSubmit } = useForm({
+    resolver: yupResolver(validationSchema),
+    defaultValues,
+  })
 
-const Home = (): JSX.Element => {
-  return <HomeTemplate />
+  const onSubmit = (): void => {
+    console.log('submit')
+  }
+  return <HomeTemplate handleSubmit={handleSubmit(onSubmit)} />
```

Comment:
Is it right this handleSubmit inside the handleSubmit param? 
Or was it supposed to be like this:
`handleSubmit={onSubmit}`